flagsmith/flagsmith.py

```diff
@@ -255,7 +267,12 @@ def _get_identity_flags_from_api(
                 return Flags(default_flag_handler=self.default_flag_handler)
             raise
 
-    def _get_json_response(self, url: str, method: str, body: dict = None):
+    def _get_json_response(
+        self,
+        url: str,
+        method: str,
+        body: typing.Optional[typing.Mapping[str, typing.Any]] = None,
+    ) -> typing.Any:
```

Comment:
1. Our SDK endpoints only return JSON objects, so it'll be fair to use a `typing.Dict[str, Any]` for the return type annotation. This will probably require a `cast` but, IMO, will improve developer experience.
2. Maybe it makes sense to define a recursive `JsonType = Union[None, int, str, bool, List[JsonType], Dict[str, JsonType]]` type and use it both for the `body` argument and the return annotation (`Dict[str, JsonType]`).

Reply:
```JsonType = Union[None, int, str, bool, List[JsonType], Dict[str, JsonType]]``` - We would have to add asserts in the current code to narrow down types as `response[key]` is failing because `response` could be a string as well. We can define a definite type for the response API as that might be helpful for developers. What do you think?

Reply:
I went ahead with using `any` as _get_json_reponse() also returns a `list` of `dicts`. Using `any` and type-casting the response later worked fine.